src/main/io/ledstrip.c

```diff
@@ -27,26 +27,55 @@
 
 #ifdef LED_STRIP
 
+#include <common/axis.h>
 #include <common/color.h>
 
 #include "drivers/light_ws2811strip.h"
 #include "drivers/system.h"
+#include "drivers/sensor.h"
+#include "drivers/accgyro.h"
 #include "drivers/serial.h"
+#include "drivers/gpio.h"
+#include "drivers/timer.h"
+#include "drivers/pwm_rx.h"
 
 #include <common/maths.h>
 #include <common/printf.h>
 #include <common/typeconversion.h>
 
+#include "flight/flight.h"
+#include "flight/failsafe.h"
+#include "flight/mixer.h"
+#include "flight/navigation.h"
+
 #include "sensors/battery.h"
+#include "sensors/sensors.h"
+#include "sensors/acceleration.h"
+#include "sensors/barometer.h"
+#include "sensors/boardalignment.h"
+#include "sensors/gyro.h"
 
-#include "config/runtime_config.h"
-#include "config/config.h"
 #include "rx/rx.h"
 #include "io/rc_controls.h"
-#include "flight/failsafe.h"
+#include "io/escservo.h"
+#include "io/gps.h"
+#include "io/gimbal.h"
+#include "io/serial.h"
+
+
+#include "config/runtime_config.h"
+#include "config/config.h"
+#include "config/config_profile.h"
+
+#include "telemetry/telemetry.h"
+
+
+
```

Comment:
There's already a local static variable which can be used. `colors`.

See `ledStripInit()`

Can you fix the code in the PR to use `colors` instead of `master_config.colors` @St7ven?